storage/sqlite.go

```diff
@@ -489,15 +501,16 @@ func (s *Sqlite) sqliteRuleToAPIKey(src *Rule) *api.Key {
 /////////////////////////////////// Types ///
 /////////////////////////////////////////////
 type User struct {
-	CreatedAt time.Time      `gorm:""`
-	UpdatedAt time.Time      `gorm:""`
-	DeletedAt gorm.DeletedAt `gorm:"index"`
-	Name      string         `gorm:"primaryKey;autoIncrement:false;notNull"`
-	Token     string         `gorm:"unique;index;notNull"`
-	Role      string         `gorm:"notNull;default:''"`
-	Readiness bool           `gorm:"notNull;default:false"`
-	Character api.Character  `gorm:"notNull;default:0"`
-	Status    api.Status     `gorm:"notNull;default:0"`
+	CreatedAt                time.Time      `gorm:""`
+	UpdatedAt                time.Time      `gorm:""`
+	DeletedAt                gorm.DeletedAt `gorm:"index"`
+	Name                     string         `gorm:"primaryKey;autoIncrement:false;notNull"`
+	Token                    string         `gorm:"unique;index;notNull"`
+	Role                     string         `gorm:"notNull;default:''"`
+	Readiness                bool           `gorm:"notNull;default:false"`
+	Character                api.Character  `gorm:"notNull;default:0"`
+	Status                   api.Status     `gorm:"notNull;default:0"`
+	LastUserStatusUpdateTime time.Time      `gorm:""`
```

Comment:
Delete "User" from the name of this argument. It's in the User structure, so the name should be "LastStatusUpdateTime"